lms/static/js/onboarding_survey/registration.js

```diff
@@ -0,0 +1,95 @@
+$(document).ready(function() {
+
+    $.ajax({
+            url: "onboarding_survey/get_user_organizations", success: function (result) {
+                full_result = result
+                orgs = Object.keys(result)
+            }
+        });
+
+        setTimeout(function () {
```

Comment:
what is the reason for this `setTimeout`?

Reply:
Removed and used this approach. https://gist.github.com/chrisjhoughton/7890303